packages/kbn-config/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@kbn/config",
+  "main": "./target/out/index.js",
```

Comment:
why we emit code and types into different folder?

Reply:
I copied the ts config of `config-schema`

https://github.com/elastic/kibana/blob/e9e0ca37820e8a2d00d5180b6357ae386d56d619/packages/kbn-config-schema/tsconfig.json#L4-L6

But we got some inconsistencies on our packages, they don't all do the same. I can change that for `config`